### ISSUE TITLE
feature_classes and django-salesforce

### ISSUE THREAD

**Issue by lociii** [laymonage/django-jsonfield-backport#24]
[django-salesforce](https://github.com/django-salesforce/django-salesforce) is an ORM integration to the Salesforce API.
It gets set up like any other database source with it's backend identifying as [vendor `salesforce`](https://github.com/django-salesforce/django-salesforce/blob/8ca907ca6b5483f2bc7a5abac772df6d648f9def/salesforce/backend/base.py#L43).

This leads to a KeyError in https://github.com/laymonage/django-jsonfield-backport/blob/56679e1999cd7d20bca1a05e6a9d793830bc72e1/src/django_jsonfield_backport/features.py#L65
because the vendor cannot be found in the `feature_classes` map.

What would be the best way to handle unsupported/unknown database vendors so that I can offer a pull request?

**Comment by laymonage** [comment]
Hi, @lociii, thanks for the report and sorry for the late reply. I think we can change that line to something like:

```python
value = feature = getattr(feature_classes.get(connection.vendor), name, False)
```

That should let the value to be `False` as a fallback for database backends not included in `feature_classes`.

**Comment by lociii** [comment]
On my way!

**Comment by laymonage** [comment]
Fixed in v1.0.3 :tada:
